myuw/templates/base.html

```diff
@@ -150,6 +151,26 @@ <h2 class="sr-only" id="app_messages_desktop_label">MyUW Announcements</h2>
         <!-- end message banner -->
         {% endblock %}
 
+	{% if display_onboard_message %}
+	{% block desktop_tour_banner %}
+        <!-- NTS: message banner based off the banner above (Desktop only) -->
+        <div class="myuw-banner-desktop app-messages" id="tour_messages_desktop" aria-labelledby="app_messages_desktop_label" role="complementary">
+            <div class="message-banner">
+                <h2 class="sr-only" id="app_messages_desktop_label">MyUW Announcements</h2>
+                <div id="message_banner_location">
+                    <div class="message-text myuw-wrapper">
+                        <div class="message">
+                            <p>New here? <a href="#" data-toggle="modal" data-target="#tour_modal">See MyUW at a glance </a></p>
+                        </div>
+                    </div>
+                </div>
+		<button id="banner_msg_close" type="button" class="myuw-banner-msg-close-btn close" aria-label="Close message" aria-controls="tour_messages_desktop"><span aria-hidden="true">&times;</span></button>
```

Comment:
Same here: `<button id="banner_msg_close"`
The ID is used for both desktop and mobile.

Reply:
mobile banner close button id is changed to id="mobile_banner_msg_close"